src/oso.service.ts

```diff
@@ -20,24 +20,20 @@ export class OsoService extends Oso implements OnModuleInit {
     });
 
     if (this.options.loadFile) {
-      await this.loadFiles([this.options.loadFile]);
+      await this._loadFiles([this.options.loadFile]);
     }
 
     if (this.options.loadFiles) {
-      await this.loadFiles(this.options.loadFiles);
+      await this._loadFiles(this.options.loadFiles);
     }
 
     if (this.options.loadStr) {
       await this.loadStr(this.options.loadStr);
     }
   }
 
-  private async loadFiles(patterns: string[]): Promise<void> {
+  private async _loadFiles(patterns: string[]): Promise<void> {
```

Comment:
Why are you adding an underscore here?

Reply:
The `loadFiles` method is from Polar's `Polar.loadFiles`, but does not load path wildcards, from what I understood. The `_loadFiles` without the underscore uses `fast-glob` to provide solution to this, but can be also renamed. Am I correct or missing something?